### PR TITLE
WIP: SL1K2 New Unit. 

### PR DESCRIPTION
**Pull request by ghalym**
## Description
Scanned new SL1K2 terminal. Updated COE parameters for motors and encoders. Updated NC parameters for SL1K2 motors.

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: lcls-plc-rixs-optics/_Config/NC/NC.xti [hunk 1]
```diff
@@ -1410,7 +1410,7 @@ External Setpoint Generation:
 			</Encoder>
 			<Drive Name="Drive" DrvType="24">
 				<DrvPara Inverse="true">
-					<Analog VeloReferenz="3.3" ScaleFactorActTorque="0.1"/>
+					<Analog VeloReferenz="0.318" ScaleFactorActTorque="0.1"/>
```
Comment (nrwslac) [inline lcls-plc-rixs-optics/_Config/NC/NC.xti:1413]: Is there somewhere to cross check these values? I don't see SL1K2 in the beckhoff motor parameter calculator sheet.

Comment (ghalym) [reply lcls-plc-rixs-optics/_Config/NC/NC.xti:1413]: These are the values we got from JJ. The system was tested and qualified with these values.
But you do make a good point. For Mechantronics grade 3 I usually just ask for scaling factor, limits and encoder counts at limits and nominals. I don't ask for gearing ration, etc. Should we in the future? Is that how the optics are delivered?

Comment (nrwslac) [reply lcls-plc-rixs-optics/_Config/NC/NC.xti:1413]: I see. I didn't realize this device came from the vendor with all these values pre-calculated. For optics we would have the basic NC settings captured in the checkout procedure on confluence at minimum; not so detailed as to include the gear ratios. I did get the ratios from Axilon for the DCCM, so I input those values in the Beckhoff calculator google sheet.

Comment (nrwslac) [reply lcls-plc-rixs-optics/_Config/NC/NC.xti:1413]: This page: https://confluence.slac.stanford.edu/display/L2SI/FAT+Controls+Checkout+-+DCCM+TXI+Monochromator+SLAC
has a motor configuration table you could copy if you want.

Comment (ghalym) [reply lcls-plc-rixs-optics/_Config/NC/NC.xti:1413]: Well during the SAT they need to demonstrate that with Beckhoff motion we can achieve all the requirements set in the ESD/SOW in terms or accuracy, repeatability, stability etc. So all the axes come tuned and tested.
Maybe we need to revisit the Mechatronics grades and add more deliverables.

**Review by nrwslac — APPROVED** [review]
lgtm.

**Review by jyotiphy — APPROVED** [review]
Looks good to me!

**Pull request merged**